scripts/a2_collections

```diff
@@ -36,8 +38,8 @@
 #     a2 -o --show users name bob
 #     a2 --v2 -o --show -p proj1,proj2 roles
 #
-#     To run on a different environment you can adjust the env vars for a single invocation. Example:
-#     TARGET_HOST=https://a2-local-fresh-install-dev.cd.chef.co TOK=stV8b5NqP6qepD9I23OFn_U2OME= a2 -s policies
+# To run on a different environment you can adjust the env vars for a single invocation. Example:
+# TARGET_HOST=https://fresh-install-dev.cd.chef.co TOK=stVpD9I23OFn_U2OME= a2 -s policies
```

Comment:
Guess it doesn't matter that we're leaking admin tokens into the world, since the machine is on our VPN. 🤷‍♂